### PR TITLE
Update email content

### PR DESCRIPTION
**Pull request by gms-gs**
Update email content

### **Due to releasing today, values have been hardcoded for both services so we can get this over the line. This will be fixed on release and we can refactor the emailing content per service in order to not cause issues**

### CODE REVIEW

File: app/controllers/claims/schools/claims_controller.rb [hunk 1]
```diff
@@ -49,7 +49,7 @@ def confirmation; end
   def submit
     return redirect_to rejected_claims_school_claim_path unless @claim.valid_mentor_training_hours?
 
-    Claims::Claim::Submit.call(claim: @claim, user: current_user)
+    Claims::Claim::Submit.call(claim: @claim, user: current_user, organisation: @school)
```
Comment (Nitemaeric) [inline app/controllers/claims/schools/claims_controller.rb:52]: `school` is accessible from the `claim` 😅

File: app/mailers/user_mailer.rb [hunk 2]
```diff
@@ -17,19 +17,25 @@ def claim_submitted_notification(user, claim)
...
+                         user_name: user.first_name,
+                         organisation_name: claim.school.name,
                          reference: claim.reference,
                          amount: claim.amount.format(symbol: true, decimal_mark: ".", no_cents: false),
+                         sign_in_url:,
                          link_to_claim:)
   end
 
-  def claim_created_support_notification(claim, email)
+  def claim_created_support_notification(claim, email, user)
```
Comment (Nitemaeric) [inline app/mailers/user_mailer.rb:28]: Here you're accepting a `user`, but in the call below, you pass through `claim.school`.

File: app/services/claims/claim/create_draft.rb [hunk 3]
```diff
@@ -22,7 +22,7 @@ def send_claim_created_support_notification_email
 
     user_emails.each do |email|
       UserMailer.with(service: :claims)
-        .claim_created_support_notification(claim, email).deliver_later
+        .claim_created_support_notification(claim, email, claim.school).deliver_later
```
Comment (Nitemaeric) [inline app/services/claims/claim/create_draft.rb:25]: Here.

File: app/services/claims/claim/submit.rb [hunk 4]
```diff
@@ -20,7 +20,7 @@ def call
 
   def send_claim_submitted_notification_email
     UserMailer.with(service: :claims)
-      .claim_submitted_notification(user, claim).deliver_later
+      .claim_submitted_notification(user, claim, claim.school).deliver_later
```
Comment (Nitemaeric) [inline app/services/claims/claim/submit.rb:23]: You don't need to change this file.

File: config/locales/en/support_user_mailer.yml [hunk 5]
```diff
@@ -5,13 +5,35 @@ en:
       body: |
         Dear %{user_name},
 
-        You have been invited to join %{service_name}.
+        You have been invited to join the %{service_name}.
```
Comment (Nitemaeric) [inline config/locales/en/support_user_mailer.yml:8]: Is this right? It reads very strangely.

"You have been invited to join the Claim funding for mentor training."

Maybe this should be?

```suggestion
        You have been invited to join the %{service_name} service.
```

File: spec/mailers/user_mailer_spec.rb [hunk 6]
```diff
@@ -10,13 +10,29 @@
 
       it "sends the invitation" do
         expect(invite_email.to).to contain_exactly(user.email)
-        expect(invite_email.subject).to eq("You have been invited to #{organisation.name}")
+        expect(invite_email.subject).to eq("Invitation to join claims funding for mentor training")
```
Comment (Nitemaeric) [inline spec/mailers/user_mailer_spec.rb:13]: This is incorrect. "claims funding for mentor training" is not our official service name.

File: app/mailers/user_mailer.rb [hunk 7]
```diff
@@ -1,14 +1,14 @@
 class UserMailer < ApplicationMailer
   def user_membership_created_notification(user, organisation)
     notify_email to: user.email,
-                 subject: t(".subject", organisation_name: organisation.name),
-                 body: t(".body", user_name: user.full_name, organisation_name: organisation.name, service_name:, sign_in_url:)
+                 subject: t(".subject", service_name: user.service),
```
Comment (Nitemaeric) [inline app/mailers/user_mailer.rb:4]: `service_name` is already available as a helper in `ApplicationMailer`. It uses looks up the official service name from our locales.

```suggestion
                 subject: t(".subject", service_name:),
```

File: app/mailers/user_mailer.rb (hunk 2 above)
Comment (Nitemaeric) [inline app/mailers/user_mailer.rb:28]: This change also makes me think, why do we need both `email` and `user`? Is `email` ever different from `user.email`?

File: app/mailers/user_mailer.rb [hunk 8]
```diff
@@ -17,19 +17,25 @@ def claim_submitted_notification(user, claim)
     notify_email to: user.email,
                  subject: t(".subject"),
                  body: t(".body",
+                         user_name: user.first_name,
+                         organisation_name: claim.school.name,
                          reference: claim.reference,
                          amount: claim.amount.format(symbol: true, decimal_mark: ".", no_cents: false),
+                         sign_in_url:,
                          link_to_claim:)
```
Comment (Nitemaeric) [inline app/mailers/user_mailer.rb:25]: This looks like it's no longer used.

**Review by Nitemaeric — APPROVED** [review]
I _think_ it's good to go now. @CatalinVoineag can you give it a final check please?

File: config/locales/en/user_mailer.yml [hunk 9]
```diff
@@ -1,33 +1,82 @@
...
     claim_created_support_notification:
-      subject: A support user created your ITT mentor training claim
+      subject: New draft claim for mentor training
       body: |
-        Reference: %{reference}
-        Amount: %{amount}
+        Dear %{user_name},
+
+        We have added a draft claim for mentor training for %{organisation_name}.
+          Your claim reference is %{reference}.
```
Comment (CatalinVoineag) [inline config/locales/en/user_mailer.yml:70]: Is there an extra tab on line 70? Does it need to be there?

**Review by CatalinVoineag — APPROVED** [review]

**Pull request merged**